lib/performance/use_case/send_to_elasticsearch.rb

```diff
@@ -11,7 +11,10 @@ def initialize(
 
   def execute
     data = @data_fetcher.fetch
-    @elasticsearch_gateway.write data
-    @logger.info "Wrote volumetrics data point '#{data}' to Elasticsearch"
+    date = Date.today.to_s
```

Comment:
This code isn't called anywhere I believe and can be deleted. The data stamp is best set in the `PerformancePlatform::Gateway::<metric>` classes I think

Reply:
Reverted this change - leaving deletion for later.